packages/react-components/src/components/MessageThread.tsx

```diff
@@ -2,12 +2,8 @@
 // Licensed under the MIT License.
 
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import { Icon, IStyle, mergeStyles, PrimaryButton, Theme, IPersona, Persona, PersonaSize } from '@fluentui/react';
-import {
-  Chat,
-  ChatMessage as FluentChatMessage,
-  ChatMyMessage as FluentChatMyMessage
-} from '@fluentui-contrib/react-chat';
+import { Icon, IStyle, mergeStyles, PrimaryButton /*, Theme, IPersona, Persona, PersonaSize*/ } from '@fluentui/react';
```

Comment:
Should we remove the commented out imports?

Reply:
Good catch!